tests/infer/mcmc/test_hmc.py

```diff
@@ -226,7 +224,7 @@ def model(data):
     assert_equal(posterior.mean, true_probs, prec=0.05)
 
 
-@pytest.mark.parametrize("jit", [False, mark_jit(True)], ids=jit_idfn)
+@pytest.mark.parametrize("jit", [True], ids=jit_idfn)
```

Comment:
this test is slow in Travis, so we only use `jit` version. We keep this pytest.mark to have a better message in Travis.

Reply:
I think given that the JIT functionality is relatively stable now, it is okay to use it as a default option without having the test name indicate it as so. I don't feel about it strongly though, so would defer to your judgment.